src/Raven.Studio/typescript/models/database/tasks/replicationAccessBaseModel.ts

```diff
@@ -63,11 +63,19 @@ class replicationAccessBaseModel {
             validation: [
                 {
                     validator: () => !this.filteringPathsRequired() || this.samePrefixesForBothDirections() || this.sinkToHubPrefixes().length,
-                    message: "Please add at least one filtering prefix path, or use the Hub to Sink prefixes."
+                    message: "Please add at least one filtering path, or use the Hub to Sink paths"
                 }
             ]
         })
     }
+    
+    private hasSingleDocumentPattern(paths: prefixPathModel[]): KnockoutComputed<boolean> {
+        return ko.pureComputed(() => paths.length && !!paths.find(x => !x.path().endsWith("*")));
+    }
+    
+    getSingleDocumentPatternWarning() {
+        return "Path patterns that do Not end with * (asterisk) will match only a single document";
```

Comment:
`do Not` ==> `do not`, no?

Reply:
fixed `Not => not` in https://github.com/ravendb/ravendb/pull/15258